src/app/legacy/containers/Header/index.test.jsx

```diff
@@ -9,7 +9,7 @@ import {
 } from '../../../components/react-testing-library-with-providers';
 import { service as pidginServiceConfig } from '../../../lib/config/services/pidgin';
 import HeaderContainer from './index';
-import { AUDIO_PAGE } from '../../../routes/utils/pageTypes';
+import { AUDIO_PAGE, TOPIC_PAGE } from '../../../routes/utils/pageTypes';
```

Comment:
Nitpick - we already have all page types imported on line 2 - could we delete this, and move AUDIO_PAGE and TOPIC_PAGE to line 21?